### PR TITLE
Add kontena/k8s-client ruby library

### PR DESCRIPTION
**Pull request by jakolehm**
To community-maintained client libraries.

### CODE REVIEW

**Comment by xiangpengzhao** [comment]
This LGTM. Thanks!
cc @bradtopol

File: content/en/docs/reference/using-api/client-libraries.md [hunk 1]
```diff
@@ -60,6 +60,7 @@ their authors, not the Kubernetes team.
 | Python               | [github.com/mnubo/kubernetes-py](https://github.com/mnubo/kubernetes-py) |
 | Ruby                 | [github.com/Ch00k/kuber](https://github.com/Ch00k/kuber) |
 | Ruby                 | [github.com/abonas/kubeclient](https://github.com/abonas/kubeclient) |
+| Ruby                 | [github.com/kontena/k8s-client](github.com/kontena/k8s-client) |
```
Comment (SpComb) [inline content/en/docs/reference/using-api/client-libraries.md]: The link is missing the `https://` prefix.

Comment (jakolehm) [reply content/en/docs/reference/using-api/client-libraries.md]: Good catch, fixed.

**Comment by bradtopol** [comment]
Great job!
/lgtm
/approve

**Pull request merged**